ontologies/works.ttl

```diff
@@ -0,0 +1,344 @@
+	rdfs:domain ww:Work ;
+	rdfs:range ww:IPRightsStatus ;
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:hasSubject rdf:type owl:ObjectProperty ;
+	rdfs:label "hasSubject"@en ;
+	rdfs:comment "Relates a work to the general thesaurus-based concept that describes the work's content."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range ww:Subject ;
+	rdf:isDefinedBy <TBC/works> .
+
+ww:hasGenre rdf:type owl:ObjectProperty ;
+	rdfs:label "hasGenre"@en ;
+	rdfs:comment "Relates a work to the genre that describes the work's content."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range ww:Genre ;
+	rdf:isDefinedBy <TBC/works> .
+	
+#######  Data properties #####  
+	
+ww:startDate rdf:type owl:DatatypeProperty ;
+	rdfs:label "startDate"@en ;
+	rdfs:comment "Relates the creation of a work to a start date, when the date of creation covers a range"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range xsd:dateTime ;
+	rdf:isDefinedBy <TBC/works> .	
+	
+ww:endDate rdf:type owl:DatatypeProperty ;
+	rdfs:label "endDate"@en ;
+	rdfs:comment "Relates the creation of a work to an end date, when the date of creation covers a range"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range xsd:dateTime ;
+	rdf:isDefinedBy <TBC/works> .		
+			
+ww:createdDate rdf:type owl:DatatypeProperty ;
+	rdfs:label "createdDate"@en ;
+	rdfs:comment "Relates the creation of a work to a date, when the date of creation does not cover a range"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range xsd:dateTime ;
+	rdf:isDefinedBy <TBC/works> .	
+
+ww:modifiedDate rdf:type owl:DatatypeProperty ;
+	rdfs:label "modifiedDate"@en ;
+	rdfs:comment "Relates the last recorded modification of a work to a date"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range xsd:dateTime ;
+	rdf:isDefinedBy <TBC/works> .	
+
+ww:identifier rdf:type owl:DatatypeProperty ;
+	rdfs:label "identifier"@en ;
+	rdfs:comment "Relates a work to an identifier such as a manuscript number"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+
+ww:title rdf:type owl:DatatypeProperty ;
+	rdfs:label "title"@en ;
+	rdfs:comment "The title or other short label of a work, including labels not present in the actual work or item but applied by the cataloguer for the purposes of search or description."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .	
+	
+ww:identifier rdf:type owl:DatatypeProperty ;
+	rdfs:label "identifier"@en ;
+	rdfs:comment "Relates a work to an identifier such as a manuscript number"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:level rdf:type owl:DatatypeProperty ;
+	rdfs:label "level"@en ;
+	rdfs:comment "Indicates the position of a work within a hierarchy of works: for example, Collection, Series, Item etc. Item in CALM descriptions is taken to mean an individual work, which is then manifested in a physical or digital item."@en ;
```

Comment:
My (admittedly limited) understanding is that our ontologies are data source agnostic – should we be mentioning CALM here?